ci-operator/config/openshift/security-profiles-operator/openshift-security-profiles-operator-main.yaml

```diff
@@ -59,6 +59,33 @@ tests:
           cpu: 100m
       timeout: 4h0m0s
     workflow: ipi-aws
+- as: e2e-flaky
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-flaky
+      cli: latest
+      commands: |
+        export E2E_CLUSTER_TYPE=openshift
+        export E2E_SPO_IMAGE=$IMAGE_FROM_CI
+        export E2E_SKIP_BUILD_IMAGES=true
+        export E2E_SKIP_NAMESPACED_TESTS=true
+        export E2E_TEST_SECCOMP=true
+        export E2E_TEST_PROFILE_RECORDING=false
+        export E2E_TEST_BPF_RECORDER=false
+        export E2E_TEST_LOG_ENRICHER=true
+        export E2E_TEST_SELINUX=true
+        export E2E_TEST_LABEL_POD_DENIALS=true
```

Comment:
This can and probably should be a separate commit (to remove the same variable from the other e2e stanza) but since the first e2e patch was merged, upstream completely removed the `label_pod_denials` feature and with it went its tests and this env variable, too.